tests/test_workers/test_tasks/test_build_create_empty_index.py

```diff
@@ -123,9 +124,94 @@ def test_handle_create_empty_index_request_raises(mock_iifbc, mock_c):
     mock_iifbc.return_value = True
     with pytest.raises(IIBError):
         build_create_empty_index.handle_create_empty_index_request(
-            from_index='index-image:latest',
+            from_index='fbc-index-image:latest',
```

Comment:
This is not a test for fbc

Reply:
this is a test to ensure IIBerror is raised when fbc=False, but from_index provided is FBC index image. Let me change the function name to make it more intuitive

Reply:
thanks

Reply:
Added a comment to describe the test as well